terratest/src/test/pega/deployment_utility.go

```diff
@@ -133,8 +133,11 @@ func VerifyDeployment(t *testing.T, pod *k8score.PodSpec, expectedSpec pegaDeplo
 	require.Equal(t, pod.Containers[0].Env[envIndex].Name, "MAX_HEAP")
 	require.Equal(t, pod.Containers[0].Env[envIndex].Value, "7168m")
 	require.Equal(t, pod.Containers[0].EnvFrom[0].ConfigMapRef.LocalObjectReference.Name, "pega-environment-config")
-
-	require.Equal(t, "2", pod.Containers[0].Resources.Limits.Cpu().String())
+    if expectedSpec.name == "pega-stream" {
```

Comment:
Instead of special casing stream to 2vCPU, let's just update it in the [values.yaml](https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/values.yaml#L163) file so it matches the other tiers at 4vCPU.

Reply:
cpuLimit: 2 was explicitly configured for stream in the values.yaml file. Should we be removing that to get the default settings? 
between values.yaml and values-large.yaml, for stream, that is one difference. Explicitly setting it in values.yaml.

Reply:
I think so - it was added there to be an example.  I just pushed an example of what I was thinking for clarity.  Feel free to modify or back it out if you would rather.

Reply:
@dcasavant if the limit was set to 2 as an example, then it make sense to remove it. Exposing it as comment is right way for people to realize such options exist.

Reply:
So effectively, we are doing the following things: 

- Setting default limit to 4
- Setting default request to 2
- Commenting the limit of 2 for stream tier in values.yaml so that i starts picking the default value of 4.
- Change the test case to not have any special cases for stream tier. 

Agree?

Reply:
I checked in a change to # 3 and 4 into your branch.  Take a look and if you are good with that, just make the change for 1 and 2.

Reply:
I will be changing to 2 and push the changes shortly

Reply:
I pushed the changes.